src/main/java/com/google/sps/servlets/FetchQuestionsServlet.java

```diff
@@ -35,19 +35,29 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** 
- * This servlet will retrieve forum posts to be displayed on the page.
+ * This servlet will retrieve questions to be displayed on the page.
  */
-@WebServlet("/fetch-forum")
-public class FetchForumServlet extends HttpServlet {
+@WebServlet("/fetch-questions")
+public class FetchQuestionsServlet extends HttpServlet {
 
   /** 
-   * This method will get the forum questions from the query and return them as a JSON string.
+   * This method will get the questions from the query and return them as a JSON string.
```

Comment:
Start method level JavaDoc with a simple present tense verb.
```suggestion
   * Gets the questions from the query and returns them as a JSON string.
```

Reply:
done